app/main/controllers/privacy/RTMediaPrivacy.php

```diff
@@ -47,9 +47,6 @@ public function __construct( $flag = true ) {
 			// show change privacy option in activity meta.
 			add_action( 'bp_activity_entry_meta', array( $this, 'update_activity_privacy_option' ) );
 
-			// To show load more button all the time.
-			add_filter( 'bp_activity_has_more_items', array( $this, 'rtm_bp_activity_has_more_items' ) );
```

Comment:
@vaishaliagola27 - Do you know why we added this previously?

Reply:
@abhijitrakas No, It was there way more before.